scripts/globals/bcnm.lua

```diff
@@ -2104,6 +2198,11 @@ xi.bcnm.onEventFinish = function(player, csid, option)
     -- player:PrintToPlayer(string.format("EventFinishBCNM csid=%i option=%i", csid, option))
     player:setLocalVar("[battlefield]area", 0)
 
+    if csid == 32000 then
+        player:setCharVar("OpenedArmouryCrate", 0)
```

Comment:
Also this setCharVar is custom to help out the module that prevents players from using force dc to skip running back to BCNM entrance.

Reply:
Removed